### DISCUSSION TITLE
Related Work Collection

### DISCUSSION THREAD

**Discussion by Kunlun-Zhu** [ulab-uiuc/research-town#44] [General]
Related work on research town

**Comment by lwaekfjlk** [comment]
https://blog.allenai.org/data-driven-discovery-with-large-generative-models-e1a062e99390
Here is one related work.

**Comment by chengzr01** [comment]
**LLM for Research** (Ideation, Experimentation, Writing, and Reviewing)

- **ResearchAgent: Iterative Research Idea Generation over Scientific Literature with Large Language Models** https://arxiv.org/pdf/2404.07738
- **MLAgentBench: Evaluating Language Agents on Machine Learning Experimentation** https://arxiv.org/pdf/2310.03302
- **Mapping the Increasing Use of LLMs in Scientific Papers** https://arxiv.org/pdf/2404.01268
- **MARG: Multi-Agent Review Generation for Scientific Papers** https://arxiv.org/pdf/2401.04259
- **ReviewerGPT? An Exploratory Study on Using Large Language Models for Paper Reviewing** https://arxiv.org/pdf/2306.00622
- **Can large language models provide useful feedback on research papers? A large-scale empirical analysis** https://arxiv.org/pdf/2310.01783
- **Monitoring AI-Modified Content at Scale: A Case Study on the Impact of ChatGPT on AI Conference Peer Reviews** https://arxiv.org/pdf/2403.07183

**Reply by lwaekfjlk** [reply]
Thanks, the list looks great!

**Reply by lwaekfjlk** [reply]
@ft2023 , here is the useful source for writing.
